wordle-web/components/SettingsDialogue.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <v-dialog v-model="modelValue" max-width="500">
+    <v-card>
+      <v-sheet color="primary">
+        <v-card-title> Settings </v-card-title>
+      </v-sheet>
+      <v-card-item>
+        <v-label>Theme</v-label>
+        <v-select
+          v-model="selctedTheme"
+          :items="['Light', 'Dark', 'Sapphire Deep Sea Dive', 'Emerald Isle','Amethyst Twilight Mist','Ruby Royale']"
+        />
+      </v-card-item>
+      <v-divider />
+
+      <v-card-actions>
+        <v-spacer></v-spacer>
+        <v-btn
+          color="secondary"
+          variant="tonal"
+          text="Apply"
+          @click="updateTheme"
+        />
+        <v-btn
+          color="secondary"
+          variant="tonal"
+          text="Close"
+          @click="modelValue = false"
+        />
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script setup lang="ts">
+const modelValue = defineModel<boolean>({ default: false });
+import { useTheme } from "vuetify";
+import nuxtStorage from "nuxt-storage";
```

Comment:
Were you ever able to get the error that would appear under 'nuxt-storage' to stop appearing? Could not figure this one out

Reply:
No i did not. It always appears as error. When we import `colors` from vuetify it also shows an issue but runs fine.